index.jsx

```diff
@@ -35,6 +36,7 @@ const Output = (editorData, style) => {
 			case 'warning': return WarningOutput(block.data, style.warning || {});
 			case 'checklist': return ChecklistOutput(block.data, style.checklist || {});
 			case 'table': return TableOutput(block.data, style.table || {});
+			case 'delimiter': return DelimiterOutput(style.delimiter || {});
 
 			default: return '';
 		}
```

Comment:
The export object needed an update...but i will take care of that :) 
Will update the NPM package soon.

Thanks for your contribution!
👍